tasks/configure_collectd.yml

```diff
@@ -115,15 +116,88 @@
   # TODO: check does the puppet tripleo::...::collectd::enable_sqlalchemy role
   # do anything besides set up the config
   # https://github.com/openstack/puppet-tripleo/blob/master/manifests/profile/base/metrics/collectd/sqlalchemy_collectd.pp
-  when: enable_sqlalchemy_collectd
+  when: tripleo_collectd_enable_sqlalchemy
   set_fact:
     todo: true
 
-# Do I need to break these into separate support tasks?
 - name: "Configure collectd to run collectd-sensubility via collectd-exec"
-  set_fact:
-    todo: true
-  when: collectd_enable_sensubility
+  when:
+    - tripleo_collectd_enable_sensubility|bool
+  block:
+    - name: Set-up sensubility config for exec
+      set_fact:
+        sensubility_exec:
+          - user: collectd
+            group: collectd
+            exec: "collectd-sensubility"
+
+    - name: Configure the exec plugin
+      set_fact:
+        collectd_plugins: "{{ collectd_plugins | union(['exec']) | unique }}"
+        collectd_plugin_exec_exec: "{{ ( collectd_plugin_exec_exec | default([]) ) | union( sensubility_exec ) }}"
+
+    - name: "Update sudoers file"
+      when:
+        - (tripleo_collectd_sensubility_exec_sudo_rule != 'nil' )|default(false)
+      block:
+        - name: "Set sudoers path"
+          set_fact:
+            sensubility_sudoers_file: "{{ collectd_conf_output_dir }}/../sudoers.d/sensubility_collectd"
```

Comment:
On puppet side the sudoers file name is composed from sensubility_ prefix and exec_user. I guess we don't need to hardcode this here, do we? IMO just sensubility is enough.

Reply:
I was trying to keep the same files as puppet generates for the most part (exceptions are the exec.conf and python.conf) 
I'll take another look at how it's generated by puppet, but can also put in some values in defaults/main so these can be overwritten, if needed.

On the testing side, I have added some molecule tests in ``molecule/enable_sensubility/verify.yml``
I'd like some feedback on whether these are sufficient, and what additional steps you'd take to verify that things were working correctly. 
I can either include them in this role, or incorporate them into infrawatch/functional-tests, which are also run in downstream jenkins, for a scenarion that includes enabling sensubility.

On testing, this role can be installed on a tripleo node and tested with [1]
OR
you can run ``molecule test --destroy never -s enable_sensubility`` and  poke around.
If you want to check in the container, then run ``molecule login -s enable_sensubility`` to easily log in.

[1] https://review.opendev.org/c/openstack/tripleo-heat-templates/+/737100